src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -3509,22 +3509,21 @@ class A
             // It does not go on to produce the other errors.
 
             compilation.VerifyDiagnostics(
-    // (33,2): error CS0246: The type or namespace name 'XDoesNotExist' could not be found (are you missing a using directive or an assembly reference?)
-    // [XDoesNotExist()]
-    Diagnostic(ErrorCode.ERR_SingleTypeNameNotFound, "XDoesNotExist").WithArguments("XDoesNotExist").WithLocation(33, 2),
-    // (34,2): error CS0181: Attribute constructor parameter 'd' has type 'decimal', which is not a valid attribute parameter type
-    // [X(1m)]
-    Diagnostic(ErrorCode.ERR_BadAttributeParamType, "X").WithArguments("d", "decimal").WithLocation(34, 2),
-    // (35,2): error CS0181: Attribute constructor parameter 'd' has type 'decimal', which is not a valid attribute parameter type
-    // [X(1)]
-    Diagnostic(ErrorCode.ERR_BadAttributeParamType, "X").WithArguments("d", "decimal").WithLocation(35, 2),
-    // (37,2): error CS0121: The call is ambiguous between the following methods or properties: 'XAttribute.XAttribute(ref int)' and 'XAttribute.XAttribute(e1)'
-    // [X(A.dyn)]
-    Diagnostic(ErrorCode.ERR_AmbigCall, "X(A.dyn)").WithArguments("XAttribute.XAttribute(ref int)", "XAttribute.XAttribute(e1)").WithLocation(37, 2),
-    // (38,2): error CS0181: Attribute constructor parameter 'd' has type 'decimal', which is not a valid attribute parameter type
-    // [X(m.NotAConstant() + 2)]
-    Diagnostic(ErrorCode.ERR_BadAttributeParamType, "X").WithArguments("d", "decimal").WithLocation(38, 2)
-                );
+                // (33,2): error CS0246: The type or namespace name 'XDoesNotExist' or 'XDoesNotExistAttribute' could not be found (are you missing a using directive or an assembly reference?)
```

Comment:
Not sure what you mean by this line exactly, but I understand the wording is not quite right, mentioning namespace.